zio/zeekio/ztests/zeek-utf8-json.yaml

```diff
@@ -0,0 +1,80 @@
+zed: "*"
+
+# These test inputs are from src/util.cc in the Zeek source code as of Jan 2022.
+# See https://github.com/zeek/zeek/blob/3d9d6e953be5ae8b72336031b6e6659222f52cea/src/util.cc#L2521
+
+# XXX PR question: do we want the funny chars to be u-escaped?  also, our code
```

Comment:
If you're asking whether we want to u-escape characters whose Go rune value exceeds `utf.RuneSelf`, then I think the answer is no.